### DISCUSSION TITLE
Allow 201 status code in client.oauthCallback?

### DISCUSSION THREAD

**Discussion by alarner** [panva/openid-client#560] [Q&A] [answered]
Hello, I'm working with an API that returns a 201 Created status code rather than a 200 status code while I'm calling the `oauthCallback`. Is there a way to configure the openid-client to allow this status code? Any help / advice much appreciated!

Thanks

**Comment by panva** [comment, marked as answer]
There is no way to work around these non-conform behaviours. Reach out to your authorization server support to have them implement the protocol as specified.
